app/controllers/api/v1/accounts_controller.rb

```diff
@@ -47,6 +47,8 @@ def follow
     options = @account.locked? || current_user.account.silenced? ? {} : { following_map: { @account.id => { reblogs: follow.show_reblogs?, notify: follow.notify?, languages: follow.languages } }, requested_map: { @account.id => false } }
 
     render json: @account, serializer: REST::RelationshipSerializer, relationships: relationships(**options)
+  rescue FollowService::SelfFollowError
+    render json: { error: I18n.t('Following your own account is not allowed') }, status: 403
```

Comment:
Using `I18n` here is good ... but it looks like this is just putting the error string right here, instead of adding an i18n key?

Reply:
Hmmm... okay so I'm not sure what adding an i18n key means. I had assumed that doing the above is what was meant. Are there any examples of using an `i18n` key that I can look at in the codebase?

Reply:
Hi! I18n strings are defined in `config/locales`. One example you can have a look at is `invites.invalid`, defined in `config/locales/en.yml` and used in `app/controllers/api/v1/invites_controller.rb`